src/static/js/mobile2.js

```diff
@@ -8,6 +8,80 @@ $(document).ready(function() {
 
         post_handler: "mobile2",
 
+        upload_animal_image: function(file, animalid, uploadtype) {
+            let reader = new FileReader();
+            reader.addEventListener("load", function() {
+                let formdata = "animalid=" + animalid + "&type=" + uploadtype + "&filename=" + encodeURIComponent(file.name) + "&filedata=" + encodeURIComponent(reader.result);
+                let targeturl =  "mobile_photo_upload";
+                $.ajax({
+                    method: "POST",
+                    url: targeturl,
+                    data: formdata,
+                    dataType: "text",
+                    error: function(e) {
+                        //console.log("Error");
+                    },
+                    success: function(mid) {
+                        let newthumbnail = '<div class="media-thumb" style="position: relative; border-style: solid; border-width: 1px; ' +
```

Comment:
a single function to render media thumbnails would make sense instead of copy/pasting this html and styling in multiple places - the only thing it needs passing is the media id and date string as far as I can tell?